app/helpers/languages_helper.rb

```diff
@@ -198,14 +198,18 @@ module LanguagesHelper
     kab: ['Kabyle', 'Taqbaylit'].freeze,
     ldn: ['Láadan', 'Láadan'].freeze,
     lfn: ['Lingua Franca Nova', 'lingua franca nova'].freeze,
+    mdf: ['Moksha', 'Мокшень кяль'].freeze,
     moh: ['Mohawk', 'Kanienʼkéha'].freeze,
+    myv: ['Erzya', 'Эрзянь кель'].freeze,
     nds: ['Low German', 'Plattdüütsch'].freeze,
     pdc: ['Pennsylvania Dutch', 'Pennsilfaani-Deitsch'].freeze,
     sco: ['Scots', 'Scots'].freeze,
     sma: ['Southern Sami', 'Åarjelsaemien Gïele'].freeze,
     smj: ['Lule Sami', 'Julevsámegiella'].freeze,
     szl: ['Silesian', 'ślůnsko godka'].freeze,
+    tlh: ['Klingon', 'tlhIngan Hol'].freeze,
```

Comment:
This is ... not in ISO-639-3..

😄

Reply:
It seems to be official as far as I can tell: https://iso639-3.sil.org/code/tlh. (Also see https://iso639-3.sil.org/about.)

Reply:
So it is!

I had looked on wikipædia and I suspect I looked at "k" not "t" page.